code/game/gamemodes/wizard/artefact.dm

```diff
@@ -880,3 +882,20 @@ var/global/list/multiverse = list()
 	heal_brute = 25
 	heal_burn = 25
 	heal_oxy = 25
+
+/mob/living/proc/skeleton_reset()
+	set name = "Offer skeleton to ghosts"
```

Comment:
I think (not sure) you need some kind of cooldown check for this, otherwise players can spam it to be supremely irksome

Also, it's a verb, so I think it should be pathed as such.

Also also, it's a `mob/living/carbon/human` level proc, so path accordingly.

Reply:
Skeleton corgi when 👀

Reply:
the issue with making it a verb is that it becomes given to everyone even when I pathed it as mob/living/carbon/human/skeleton I have no idea how to work around this and it would just be easier to keep it as a proc that only wizard-revived skeletons can get

Reply:
Ahh, fair enough!